Kek.js

```diff
@@ -12,13 +12,14 @@ const JOSE_ALGORITHM_MAP = {
 export class Kek {
   /**
    * Creates a new instance of a key encryption key.
+   * Used to protect the master encryption key e.g. Controller Key.
```

Comment:
The `ControllerKey` is not a content encryption key, suggest:

```suggestion
   * Used to protect the content encryption key.
```

Also ... please fix the whitespace issue here like the others.